aries-site/src/pages/components/spinner.mdx

```diff
@@ -0,0 +1,71 @@
+import { Example } from '../../layouts';
+import {
+  AnnounceSpinnerExample,
+  SpinnerExample,
+  ListSpinnerExample,
+  ContentSpinnerExample,
+} from '../../examples';
+
+<Example
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/spinner/SpinnerExample.js"
+  docs="https://v2.grommet.io/spinner?theme=hpe"
+  figma="https://www.figma.com/file/feIeYk1zX36y9E7e7rprc2/HPE-Loading-Indicator-Component?node-id=0%3A1"
+>
+  <SpinnerExample />
+</Example>
+
+## Guidance
+
+Loading Spinners are used to notify the user that an action is being processed. This reassures the user of the system status. 
+
+Loading spinners can be presented in various sizes depending on the context in which the spinner is being used.
+
+For fast processes (less than 300ms), a spinner may not be required. However, for a process that may take time, a spinner should be used.
+
+### When to use Spinner
+
+- When fetching data or content, where the expected response time is greater than ~300ms.
+- When loading more information into a Table or List.
+
+### Accessibility
+
+For additional clarity for screen reader users, provide a [message](https://v2.grommet.io/spinner#message) to the spinner that would explain to the user what kind of content is being loaded.
```

Comment:
```suggestion
For additional clarity for screen reader users, provide a [message](https://v2.grommet.io/spinner#message) to the spinner that would explain to the user that the content had been loaded.
```
We can also add the support for a message as an object of `{start: ... , end: ....}` that allows announcing a message when the component had loaded and when it finished loading. https://v2.grommet.io/spinner#message

Reply:
Sounds good I will update the copy.